tests/e2e/specs/NestedScreen.js

```diff
@@ -0,0 +1,32 @@
+describe('Test Nested screen', () => {
+  it('With a variable defined', () => {
+    cy.visit('/');
+    cy.get('#controls > div:contains("Nested Screen")').drag('#screen-container div:contains("Drag an element here")', 'bottom');
```

Comment:
Consider using more declarative selectors in your tests. Tags, class names and IDs can change any time and cause your tests to fail even though the functionality hasn't changed. Have a read through https://docs.cypress.io/guides/references/best-practices.html#Selecting-Elements.

Reply:
Fixed